updating/updating-restricted-network-cluster/mirroring-image-repository.adoc

```diff
@@ -0,0 +1,171 @@
+:_content-type: ASSEMBLY
+[id="mirroring-ocp-image-repository"]
+= Mirroring the OpenShift Container Platform image repository
+include::_attributes/common-attributes.adoc[]
+:context: mirroring-ocp-image-repository
+
+toc::[]
+
+You must mirror container images onto a mirror registry before you can update a cluster in a disconnected environment. You can also use this procedure in connected environments to ensure your clusters only use container images that have satisfied your organizational controls on external content.
+
+[NOTE]
+====
+This registry must be running at all times as long as the cluster is running.
+====
+
+There are two supported methods for mirroring images onto a mirror registry:
```

Comment:
Do we need to use `supported` explicitly here?

Reply:
I guess not, I can drop it if it's okay to just say "there are two methods for..."